### DISCUSSION TITLE
UGS (Universal gcode sender error message): java.lang.NullPointerException

### DISCUSSION THREAD

**Discussion by shenzhenadam** [winder/Universal-G-Code-Sender#2508] [Help]
UGS (Universal gcode sender error message): java.lang.NullPointerException
Anyone else had this error message: java.lang.NullPointerException: Cannot invoke "javax.swing.text.AttributeSet.getAttribute(Object)??

I see previously there has been a slightly similar issue .. but not specifically related to mine.

It happens when uploading gcode - and then the error message appears and the start button is all greyed out. See images below. If anyone wants to see the complete text in the details, I can post.

I don't have this issue if using CNCJS (although it still does not follow the path on dry run) so I got no idea what the issue is. I am using Windows 11 home edition on a mini i5 pc which should be no issue unless there are some unknowns. I'm using the latest version of UGS with Fluidnc and 6Xcnc controller from same company. Is it to do with the .yaml file?  ... attached below as text file.

I have also installed the latest version of Java, and that did not make any difference.

Any advice or suggestions please?

Many thanks!
![IMG_20240419_232258 (Large)](https://github.com/winder/Universal-G-Code-Sender/assets/54853892/ac8c1470-c572-4f52-a992-67bf4587bbf8)
![IMG_20240419_232231 (Large)](https://github.com/winder/Universal-G-Code-Sender/assets/54853892/e877ba3e-d62e-4bed-ab22-732fb3d8a720)

[6x_default-adam-XYYZ-200424.txt](https://github.com/winder/Universal-G-Code-Sender/files/15047373/6x_default-adam-XYYZ-200424.txt)

**Comment by breiler** [comment]
Can you post your log file:
`C:/Users/[your username>]/AppData/Roaming/ugsplatform/var/log/messages.log`

**Reply by shenzhenadam** [reply]
Hello .. Thanks .. Please see attached files ... There are three log files - So I  have added all.
[messages.log.1.txt](https://github.com/winder/Universal-G-Code-Sender/files/15048431/messages.log.1.txt)
[messages.log.2.txt](https://github.com/winder/Universal-G-Code-Sender/files/15048432/messages.log.2.txt)
[messages.log.txt](https://github.com/winder/Universal-G-Code-Sender/files/15048436/messages.log.txt)

**Reply by breiler** [reply]
It looks like the program has been moved from `C:\Users\CNC Machine\Downloads\win64-ugs-platform-app-2.1.6(1)\ugsplatform-win\` to `D:\CNC 16 Apr 2024\win64-ugs-platform-app-2.1.6\` and has somehow gotten corrupted. Try and delete the settings under the directory `C:\Users\CNC Machine\AppData\Roaming\ugsplatform` and restart UGS

**Reply by shenzhenadam** [reply]
I deleted ugsplatform folder in roaming & local and restarted - it's OK now ..
Thanks for your help!
